docker-compose.yml

```diff
@@ -1,33 +1,25 @@
 db:
-  build: db
+  image: mattermost/mattermost-prod-db
   volumes:
     - ./volumes/db/var/lib/postgresql/data:/var/lib/postgresql/data
-    - /etc/localtime:/etc/localtime:ro
+  environment:
+    - POSTGRES_PASSWORD
   # uncomment the following to enable backup
   #environment:
   #  - AWS_ACCESS_KEY_ID=XXXX
   #  - AWS_SECRET_ACCESS_KEY=XXXX
   #  - WALE_S3_PREFIX=s3://BUCKET_NAME/PATH
   #  - AWS_REGION=us-east-1
 app:
-  build: app
+  image: mattermost/mattermost-prod-app
   links:
-    - db:db
+    - db
   volumes:
-    - ./volumes/app/mattermost/config:/mattermost/config:rw
-    - ./volumes/app/mattermost/data:/mattermost/data:rw
-    - /etc/localtime:/etc/localtime:ro
+    - ./volumes/app/mattermost/config:/opt/mattermost/config
+    - ./volumes/app/mattermost/data:/opt/mattermost/data
+  environment:
+    - POSTGRES_PASSWORD
 web:
-  build: web
-  ports:
-    - "80:80"
-    - "443:443"
+  image: mattermost/mattermost-prod-web
```

Comment:
We can't use the images on dockerhub which are not updated yet. This breaks the travis test.